accepted/0000-timezones.md

```diff
@@ -0,0 +1,98 @@
+- Feature Name: timezones
+- Start Date: 2019-09-16
+- RFC PR: 
+
+# Summary
+[summary]: #summary
+
+Timezones can be a big problem when managing an application like Uyuni that can be deployed and accessed from anywhere in the world. 
+If we don’t have some rules on how to display and handle dates, this can become quite a big mess to manage. 
+
+
+# Motivation
+[motivation]: #motivation
+
+The goal of this RFC is to define those rules and serve as a reference for the future.
+
+These rules should define a consistent way of display dates on the UI and make this information as clear as possible for the user.
+
+The user should be able to easily understand in which timezone a date is displayed and aware if there is any difference from the timezone
+he is accessing from.
+
+# Glossary
+[glossary]: #glossary
+
+When talking about uyuni there are 3 different Timezones contexts that are important to understand and distinguish:
+
+* *Server timezone:* the configured timezone in the server where uyuni is deployed 
```

Comment:
I this this is the timezone of the Uyuni Server. 
I think we should also add *managed system timezone* here. Even if they play no role in this rfc.